webextensions/api/management.json

```diff
@@ -403,6 +403,33 @@
                 }
               }
             }
+          },
+          "install": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
```

Comment:
@Elchi3 If Android and Webview are missing, will the rendered table have 'No' or '?'? I've forgotten.

Reply:
Question mark

Reply:
The rendered table will just omit those columns (at least in the WebExtension docs) like: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads/download#Browser_compatibility.